roles/pulp/defaults/main.yml

```
@@ -0,0 +1,11 @@
+---
+pulp_image: pulp/pulp:stable
+pulp_ports:
+  - "8080:80"
```

```suggestion
pulp_port: "8080:80"
```

`PublishPort` does not accept an array in the Quadlet file. Which if you were trying to look at https://docs.ansible.com/ansible/latest/collections/containers/podman/podman_container_module.html and map it, the Ansible module abstraction does allow you to define an array of ports, however, in that case it's looping over the array to create multiple `PublishPort` entries. Similar to what you did with the volumes.

---

That's interesting, thanks for sharing. Updated it.